### PR TITLE
Python script to generate pmc and commiter list

### PR DESCRIPTION
**Pull request by terbolous**
Made a small tool to create pmc and committer list by parsing the JSON
files on projects.apache.org.
This is accessible by anyone and does not require any ASF privileges.

Usage: python makewholist.py

Only dependencies are json, urllib and sys, which should be included by
default on most systems.

### CODE REVIEW

**Comment by terbolous** [comment]
As mentioned in #13, I went ahead and made a script that does not rely on any ASF privileges.

File: makewholist.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+groups_url = "https://projects.apache.org/json/foundation/groups.json"
+names_url = "https://projects.apache.org/json/foundation/people_name.json"
+
+names = {}   # Placeholder for names parsed in getNames()
+groups = {}  # Placeholder for groups parsed in getCommittersAndPMC()
+
+def getCommittersAndPMC():
+    global groups
+    try:
+        response = urllib.urlopen(groups_url)
```
Comment (sedukull) [inline makewholist.py]: mention global for groups_url as well.

File: makewholist.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        groups = json.loads(response.read())
+    except Exception, e:
+        print "Error during groups JSON download or parsing: %s" % e.message
+
+def getNames():
+    global names
+    try:
+        response = urllib.urlopen(names_url)
```
Comment (sedukull) [inline makewholist.py]: use global names_url here as well.

File: makewholist.py (hunk 1 above)
Comment (terbolous) [inline makewholist.py]: It is not really necessary to use global to read variables, is there a reason you want it to be?

**Comment by wido** [comment]
LGTM

**Comment by karuturi** [comment]
:+1:

**Comment by sedukull** [comment]
Was looking to be consistent when referring to global variables, but otherwise LGTM.

**Pull request merged**